python/triqs_dft_tools/converters/plovasp/proj_group.py

```diff
@@ -416,8 +416,9 @@ def orthogonalize_projector_matrix(self, p_matrix):
         overlap = np.dot(p_matrix, p_matrix.conj().T)
 # Calculate [O^{-1/2}]_{m m'}
         eig, eigv = np.linalg.eigh(overlap)
-        assert np.all(eig > 0.0), ("Negative eigenvalues of the overlap matrix:"
-           "projectors are ill-defined")
+        eig = np.around(eig,10)  
+        #assert np.all(eig > 0.0), ("Negative eigenvalues of the overlap matrix:"
```

Comment:
I guess for NC calculations this check is not useful, but we should keep it for normal calculations I guess?